server/graphql/common/expenses.ts

```diff
@@ -315,8 +315,12 @@ export const canSeeExpenseDraftPrivateDetails: ExpensePermissionEvaluator = asyn
 };
 
 /** Checks if the user can verify or resend a draft */
-export const canVerifyDraftExpense: ExpensePermissionEvaluator = async (req, expense) => {
-  return remoteUserMeetsOneCondition(req, expense, [isOwner, isCollectiveAdmin, isHostAdmin]);
+export const canVerifyDraftExpense: ExpensePermissionEvaluator = async (req, expense): Promise<boolean> => {
```

Comment:
Just keep in mind that this is shared between two different use cases.
- FromCollectiveId, after submitting the draft can verify it.
- Owner, CollectiveAdmin, and HostAdmin can resend the draft.

The logic that automatically validates the expense once the user confirms their email lives in the frontend, perhaps we should split this into two different permissions in the future.

Reply:
The shortcut you took to keep single permission for both use cases makes sense; it simplifies things.